app/next-client-app/api/datasets.ts

```diff
@@ -15,6 +15,7 @@ const fetchKeys = {
     dataset ? `projects/?dataset=${dataset}` : "projects/",
   updateDataset: (id: number) => `datasets/update/${id}/`,
   permissions: (id: string) => `dataset/${id}/permissions/`,
+  createDataset: "datasets/create/",
```

Comment:
```suggestion
  create: "datasets/create/",
```
When we're in the `dataset` namespace, so fine to name things shorter.